crates/fluvio-storage/src/iterators.rs

```diff
@@ -5,9 +5,9 @@ use std::io::{Error as IoError, ErrorKind, Cursor};
 
 use nix::sys::uio::pread;
 
+use fluvio_compression::CompressionExt;
```

Comment:
why needed here?

Reply:
Because we have some compression logic going on here: https://github.com/infinyon/fluvio/pull/4193/files/7441509eae14aa0900cfaf68d81afd07e9dc4350#diff-eedcb248ce7c084703163bcd1f43eb4595ed0af4bdaf068a2f5bc7b9228cc7e4R146